### PR TITLE
Added new things and some correction.

### PR DESCRIPTION
**Pull request by meeeaCH**
- Added R36S platform
- Added Nintendo DS to the systems
- Added a few games to the database
- Corrected some things (1 line in README, 1 line in a game)

### CODE REVIEW

File: tools/helpers/templates.py [hunk 1]
```diff
@@ -16,6 +16,10 @@ def generate_game_templates_md(game_name, normalized_system_name):
         "psp": (
             "## {game_name}\n\n%game_overview%\n\n# Execution information\n\n"
             "**Tester**:\n**Backend**:\n**Resolution**:\n**Frameskip**:\n**Autoframeskip**:\n**Note**:"
+        ),
+        "nds": (
+            "## {game_name}\n\n%game_overview%\n\n# Execution information\n\n"
+            "**Tester**:\n**High-Reslution 3D**:\n**Frame Skip Type**:\n**Frame Skip Value**:\n**Note**:"
```
Comment (ogregorio) [inline tools/helpers/templates.py]: High-Resolution*

File: platforms/tsp/systems/nds/soniccolors/soniccolors.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Sonic Colors
+
+%game_overview%
+
+# Execution information
+
+**Tester**: meeeaCH
+**High-Reslution 3D**: ON
```
Comment (ogregorio) [inline platforms/tsp/systems/nds/soniccolors/soniccolors.md]: High-Resolution*

File: platforms/tsp/systems/nds/marioluigipartnersintime/marioluigipartnersintime.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+## Mario & Luigi: Partners in Time
+
+%game_overview%
+
+# Execution information
+
+**Tester**: meeeaCH
+**High-Reslution 3D**: ON
```
Comment (ogregorio) [inline platforms/tsp/systems/nds/marioluigipartnersintime/marioluigipartnersintime.md]: High-Resolution*

File: platforms/tsp/systems/nds/mariopartyds/mariopartyds.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+## Mario Party DS
+
+%game_overview%
+
+# Execution information
+
+**Tester**: meeeaCH
+**High-Reslution 3D**: ON
```
Comment (ogregorio) [inline platforms/tsp/systems/nds/mariopartyds/mariopartyds.md]: High-Resolution*

File: platforms/r36s/index.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "R36S",
+    "database_key": "r36s",
+    "manufacturer": "-",
```
Comment (ogregorio) [inline platforms/r36s/index.json]: I think the manufacturer is Boyhom

File: platforms/r36s/index.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    "arch": "aarch64",
+    "storage": "microsSD only, 16G-256G",
+    "media": "Digital download",
+    "connectivity": [
+        "2 USB-C",
+        " 3.5 Jack"
+    ],
+    "systems": [
+        {
+            "name": "Name",
```
Comment (ogregorio) [inline platforms/r36s/index.json]: Are the systems empty for now, as they are a work in progress?

File: platforms/index.json [hunk 7]
```diff
@@ -5,6 +5,12 @@
             "image": "platforms/tsp.webp",
             "database_key": "tsp",
             "system": "Linux"
+        },
+        {
```
Comment (ogregorio) [inline platforms/index.json:10]: Add attribute image: platforms/r36s.webp
```
{
            "name": "R36S",
            "database_key": "r36s",
            "image": "platforms/r36s.webp",
            "system": "Open source Linux system"
}
```

**Review by ogregorio — CHANGES_REQUESTED** [review]

**Comment by ogregorio** [comment]
I added some comments

File: platforms/r36s/index.json (hunk 5 above)
Comment (meeeaCH) [reply platforms/r36s/index.json]: I tried to Google it, but did not find any definitive answer. I'll check it out tomorrow too, if I don't find anything, I'll set it to Boyhum.

File: platforms/r36s/index.json (hunk 6 above)
Comment (meeeaCH) [reply platforms/r36s/index.json]: It seems there is a bug in the code, I added NDS to TSP too, but it's missing.

**Comment by meeeaCH** [comment]
I think I fixed what needed to be fixed. And made some more changes. 
I tested the script and it seems to work fine, please test it to make sure. For example: R36S could have GBA system, I just put some game on my device.

I'll add a few games to it so it's not empty.

**Comment by ogregorio** [comment]
Nice additions, thank you. I will be migrating the repository to the Handheld-Database organization!

https://github.com/Handheld-Database/

**Pull request merged**